website/docs/integrate/integration-guide.md

```diff
@@ -45,10 +45,10 @@ For more information, please see the [CKB JSON-RPC protocols](https://github.com
 ### How to manage cells(like UTXO set)
 Cell management mainly affects cell collection and address balance display. There are many ways to manage cells; here are two typical example solutions.
 
-* **Recommended**: use [CKB Indexer](https://github.com/nervosnetwork/ckb-indexer) to collect cells and display balance.
-  * [get_cells_capacity](https://github.com/nervosnetwork/ckb-indexer#get_cells_capacity) Returns the live cells total capacity by the lock or type script. You can use this RPC to display address balance.
+* **Recommended**: use CKB indexer to collect cells and display balance. Recommend using [built-in](#ckb-built-in-indexer) version available since v0.105.0, [standalone](https://github.com/nervosnetwork/ckb-indexer) version is also available.
```

Comment:
```suggestion
* **Recommended**: use CKB indexer to collect cells and display balance. Recommend using [built-in](#ckb-built-in-indexer) version available since v0.106.0, [standalone](https://github.com/nervosnetwork/ckb-indexer) version is also available.
```

CKB v0.105.x was yanked.